.github/workflows/elixir.yml

```diff
@@ -2,56 +2,59 @@ name: Elixir CI
 
 on:
   push:
-  pull_request:
     branches:
       - main
-
-env:
-  ELIXIR_LATEST: '1.16.0'
```

Comment:
What's the meaning of this change?

Reply:
Only run CI on `push` even to `main` branch or `pull_request`.

From:

![image](https://github.com/user-attachments/assets/9a128171-5df7-4597-b42a-57dd79269e63)

To:

![image](https://github.com/user-attachments/assets/19a966a0-5360-44b3-88b2-1a0c4e5ae005)